Traceback (most recent call last):
  File "./convert-cn-to-skos.py", line 72, in <module>
    recs = oai.listRecords(metadataPrefix='marc21', set='corporateNames')
  File "/usr/lib/python2.6/site-packages/oaipmh/common.py", line 115, in method
    return obj(self, **kw)
  File "/usr/lib/python2.6/site-packages/oaipmh/common.py", line 110, in __call__
    return bound_self.handleVerb(self._verb, kw)
  File "/usr/lib/python2.6/site-packages/oaipmh/client.py", line 65, in handleVerb
    kw, self.makeRequestErrorHandling(verb=verb, **kw))    
  File "/usr/lib/python2.6/site-packages/oaipmh/client.py", line 273, in makeRequestErrorHandling
    raise error.XMLSyntaxError(kw)
oaipmh.error.XMLSyntaxError: {'verb': 'ListRecords', 'set': 'corporateNames', 'metadataPrefix': 'marc21'}
